pkg/monitoring/domainstats/prometheus/prometheus.go

```diff
@@ -556,11 +588,11 @@ func (ps *prometheusScraper) Scrape(socketFile string, vmi *k6tv1.VirtualMachine
 		return
 	}
 
-	ps.Report(socketFile, vmi, vmStats)
+	ps.Report(socketFile, vmi, vmDomainStats, vmFSStats)
 }
 
-func (ps *prometheusScraper) Report(socketFile string, vmi *k6tv1.VirtualMachineInstance, vmStats *stats.DomainStats) {
-	// statsMaxAge is an estimation - and there is not better way to do that. So it is possible that
+func (ps *prometheusScraper) Report(socketFile string, vmi *k6tv1.VirtualMachineInstance, vmDomainStats *stats.DomainStats, vmFSStats k6tv1.VirtualMachineInstanceFileSystemList) {
```

Comment:
@machadovilaca I think it could be better if you will create a `type vmStats struct` and you will be able to encapsulate the `vmDomainStats` and `vmFSStats` there

Reply:
I like it, changed